scripts/quota_log.py

```diff
@@ -35,12 +47,19 @@ def disk_quota():
 
 def cpu_hours():
     current_time = datetime.datetime.now()
-
-    # script that runs on uppmax
-    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    try:
+        # script that runs on uppmax
+        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
```

Comment:
@Galithil / @pekrau / anyone else - any thoughts on using `exit()`? I have in the back of my mind that it's bad practice for some reason, but I have no idea why. I think I would probably use `os.exit(1);` (exit in an error state)